Embedded/MaxMix/Config.h

```diff
@@ -49,6 +49,13 @@ static const uint8_t  STATE_DISPLAY_SLEEP = 1;
 // --- Display
 static const uint8_t  DISPLAY_RESET =   4; // Reset pin # (or -1 if sharing Arduino reset pin)
 
+// --- LCD Config
+#define SCREEN_LCD;
+static const uint8_t AMOUNT_OF_COLS = 16; // How many possible columns in the LCD
+static const uint8_t AMOUNT_OF_ROWS = 4;  // How many possible rows in the LCD
+static const uint8_t RS = 7, EN = 8, D4 = 9, D5 = 10, D6 = 11, D7 = 12; //Pins on Arduino to LCD
```

Comment:
So these pins are actually not accessible by when mounted in the enclosure (see #124 ), but either way, this whole PR requires rework of the enclosure and adds new hardware. Just a note.